Yona/Yona/SignUp/SignUpFirstStepViewController.swift

```diff
@@ -19,6 +19,7 @@ class SignUpFirstStepViewController: UIViewController,UIScrollViewDelegate {
     @IBOutlet var personalQuoteLabel: UILabel!
     @IBOutlet var scrollView: UIScrollView!
     @IBOutlet var nextButton: UIButton!
+    let extraSpaceForView = 20.0
```

Comment:
Is it necessary? Hardcoded values, related to the UI, without any comments, could be very dangerous to maintain.